sms_toolkit/messages/truncate.py

```diff
@@ -0,0 +1,59 @@
+from sms_toolkit import utils
+from sms_toolkit import constants
+from .normalization import normalize_to_unicode
+
+
+def truncate_message(message, max_length, is_for_mms=False):
+    """
+    Truncate the given SMS message string upto the provided length. If the SMS message length is less than
+    the length, or if the length is invalid (< 1), then the message is returned as is
+
+    Args:
+        message (basestring): The raw message string to profile.
+        max_length (int): The length upto which the message must be truncated. Must be greater than 0
+        is_for_mms (bool): Is this message for mms sending.
+
+    Returns:
+        (Text): The truncated message
+    """
+    if max_length < 1:
+        return normalize_to_unicode(message)
+
+    if is_for_mms:
+        encoding = constants.UCS2_ENCODING
+    else:
+        encoding = utils.determine_encoding_for_string(message)
+
+    if encoding == constants.UCS2_ENCODING:
+        truncated_message = _perform_truncation(
+            message=message,
+            max_length=max_length * 2,
```

Comment:
Whats the story with the 2x multiplier here? Can we move that to a constant with a descriptive name to help clear up what it is used for?

Reply:
Yeah for all UCS2 operations, we've used the 2 multiplier across the codebase. I can add a comment here about why that is. Not super sure though about what to move into the constants.